atlasdb-dbkvs-tests/src/test/java/com/palantir/atlasdb/keyvalue/dbkvs/DbkvsPostgresTestSuite.java

```diff
@@ -79,6 +79,7 @@ public static DbKeyValueServiceConfig getKvsConfig() {
                 .dbPassword(ImmutableMaskedValue.of("palantir"))
                 .host(postgresAddress.getHostName())
                 .port(postgresAddress.getPort())
+                .maxConnections(1)
```

Comment:
But we get better correctness testing to make sure we don't write more deadlocks. I understand the concern, but I kind of insist on this. Maybe we could discuss this with a larger audience?